packages/core/content-releases/admin/custom.d.ts

```diff
@@ -0,0 +1,21 @@
+export {};
```

Comment:
I see this file being defined practically in every package, have you discussed about adding the type into a single place?

Reply:
I think we discussed this when migrating the Helper Plugin, and yes, definitely we need to find a place to do this only one time